packages/zero-client/src/client/zero.ts

```diff
@@ -562,6 +568,7 @@ export class Zero<
       slowMaterializeThreshold,
       assertValidRunOptions,
     );
+    this.queryDelegate = this.#zeroContext;
```

Comment:
I'm a little bit concerned about exposing things that we do not expect people to use.